__init__.py

```diff
@@ -162,6 +162,13 @@ def _handle_client_exception(self, callback, *args, **kwargs):
         return False
 
     # Intent handlers
+    @intent_handler('camera.intent')
```

Comment:
Is the idea that there will only be a single `camera.intent` or is there likely to be more?
If there will be more, then we should consider renaming this to represent the specifics of this intent a little more, eg `camera.show.image.intent`?

Reply:
I used `show.camera.image.intent` to be consistent with the other namings of the plugin.